### DISCUSSION TITLE
Java serilog connects to seq but does not log

### DISCUSSION THREAD

**Discussion by Timtendo12** [datalust/seq-tickets#1915] [Help]
So I'm trying to set up a serilogger to log to our Seq instance, It seems like it can connect to Seq but when I log something it never shows up in Seq itself. The selfLogger also is empty. I'm using Serilogj version 0.6.1

```java
import serilogj.Log;
import serilogj.LoggerConfiguration;
import serilogj.debugging.SelfLog;
import serilogj.events.LogEventLevel;
...

public void initSeriLogger(String seqAPIUrl, String seqAPIKey) {

        try {
            // create a file to store the log of SelfLog

            File logFile = new File(getDataFolder(), "serilog.log");

            if (!logFile.exists()) {
                if(!logFile.createNewFile()) {
                    throw new IOException("Failed to create log file.");
                }
            }

            PrintStream output = new PrintStream(logFile);

            // set the log file as the log of SelfLog
            SelfLog.setOut(output);

            Log.setLogger(new LoggerConfiguration()
                    .writeTo(seq(seqAPIUrl, seqAPIKey))
                    .setMinimumLevel(LogEventLevel.Warning)
                    .createLogger());

        } catch (Exception e) {
            Core.getPlugin().getLogger().warning("!!!!!Serilog failed to initialize. Logging to console only.!!!!");
            return;
        }

        this.getLogger().info("Serilog initialized. Logging to '" + seqAPIUrl + "' with API key ending with '" + seqAPIKey.substring(0, 4) + "'.");
        Log.warning("Serilog initialized.");
    }
```

### console output:

`[21:21:47 INFO]: [Core] Serilog initialized. Logging to 'http://{myserverip}:5341' with API key ending with 'HPtJ'.`

### Ingestion log output:

`[2023-07-17T18:01:13.3658359+00:00 DBG] The ingestion API is configured and accepting requests`

### Seq being empty:
<img width="1370" alt="vX8D1" src="https://github.com/datalust/seq-tickets/assets/41442518/1b12b9ef-aed4-49a0-a58f-fa6471c0b83a">

Anyone knows what exactly I'm doing wrong?

**Comment by liammclennan** [comment]
Hi Tim,
You have serilog configured to only send warnings. Your warning event does not appear in any of your outputs. Perhaps `this.getLogger().warning(...)`?
